src/components/VSelect/mixins/select-autocomplete.js

```diff
@@ -58,22 +58,52 @@ export default {
         i, this.searchValue, this.getText(i))
       )
     },
+    onTabDown (e) {
+      if (this.tags) {
+        const newItem = this.filteredItems.length && this.$refs.menu.listIndex >= 0
+          ? this.filteredItems[this.$refs.menu.listIndex]
+          : this.searchValue
+        if (newItem) {
+          e.preventDefault()
+          this.addTag(newItem)
+        } else {
+          this.blur()
+        }
+        return
+      }
+
+      if (!this.menuIsActive || !this.isAutocomplete || this.$refs.menu.listIndex === -1) {
+        this.blur()
+        return
+      }
+
+      if (this.menuIsActive && this.filteredItems.length && this.$refs.menu.listIndex > -1) {
+        e.preventDefault()
+        this.$refs.menu.tiles[this.$refs.menu.listIndex].click()
```

Comment:
Would it be possible for this to fail if the menu tiles were dynamically updated and not available at the specific index?